### PR TITLE
kiss: ignore libraries with RPATH in dependency detector

### PR DESCRIPTION
**Pull request by git-bruh**

### CODE REVIEW

**Comment by git-bruh** [comment]
@illiliti

File: kiss [hunk 1]
```diff
@@ -713,18 +725,16 @@ pkg_fix_deps() {
...
         done <<EOF
 $elf
 EOF
 
         # The readelf mode requires ldd's output to resolve the library
         # path for a given file. If ldd fails, silently skip the file.
-        # We set LD_LIBRARY_PATH to the RPATH we found above to allow ldd
-        # to resolve accurate paths.
-        ldd=$(LD_LIBRARY_PATH=$(prepend "$lib_rpath" : "$pkg_dir/$repo_name") ldd -- "$pkg_dir/$repo_name$_file" 2>/dev/null) || continue
+        ldd="$(ldd -- "$pkg_dir/$repo_name$_file" 2>/dev/null)" || continue
```
Comment (illiliti) [inline kiss:737]: Remove quoting: ""

File: kiss [hunk 2]
```diff
@@ -713,18 +725,16 @@ pkg_fix_deps() {
                 value=${value##*\[}
                 value=${value%%\]*}
 
-                lib_rpath="$value"
-                break
+                should_ignore_rpath "$value" && continue 2
+                break # Found RPATH which shouldn't be ignored
```
Comment (illiliti) [inline kiss:729]: kiss doesn't have inline comments. we should keep it that way

File: kiss [hunk 3]
```diff
@@ -647,21 +647,23 @@ pkg_strip() {
...
     # Intentional, globbing disabled.
     # shellcheck disable=2086
-    { IFS="$sep"; set -- $1; unset IFS; }
-
-    for str do
-        ok "$str" || continue
-        cat="$cat$sep$pre$str"
+    { IFS=:; set -- $1; unset IFS; }
+
+    for path; do
```
Comment (illiliti) [inline kiss:655]: Remove `;`

**Review by illiliti — CHANGES_REQUESTED** [review]

**Comment by git-bruh** [comment]
Oh I missed your replies to my messages on IRC somehow. The other way is to get dirname of the interpreter and use that as the default search path but that's as unportable as this solution is

Also I feel it doesn't matter that much cuz let's say /usr/lib or /lib aren't in the default search path. Then it's quite likely that nothing will ever be installed there either

**Pull request merged**